clients/cobol-lsp-vscode-extension/src/services/copybook/LocalCopybookResolver.ts

```diff
@@ -32,10 +32,12 @@ export class LocalCopybookResolver implements CopybookResolver {
     private resolveURIList(list: string[]): string[] {
         const result: Set<string> = new Set<string>();
         list.filter(resource => resource !== "*").forEach(resource => {
-            const URI: URL = getURIFromResource(resource);
-            if (URI !== undefined) {
-                result.add(URI.href);
-            }
+            const uris: URL[] = getURIFromResource(resource);
+            uris.forEach(uri => {
+                if (uri != undefined) {
```

Comment:
Can it actually be undefined?

Reply:
have no idea, I'd rather not to change it in this PR